src/ir/table-utils.cpp

```diff
@@ -21,6 +21,26 @@
 
 namespace wasm::TableUtils {
 
+bool isSubType(const Table& a, const Table& b) {
+  if (a.addressType != b.addressType) {
+    return false;
+  }
+
+  if (!Type::isSubType(a.type, b.type)) {
+    return false;
+  }
+
+  if (a.initial > b.initial) {
+    return false;
+  }
+
+  if (a.max < b.max) {
+    return false;
+  }
```

Comment:
Are these correct? They look like they're the opposite of the memory subtyping conditions.

Reply:
Good catch, looks like I got it backwards at the call site as well. Fixed now.